company/careers.md

```diff
@@ -0,0 +1,54 @@
+# Careers
```

Comment:
Can we put the EEO text on this page as well?

Reply:
That wasn't in the careers repo README. Where can I find that copy to add?

Reply:
It's the text at the bottom of all of the job listings saying we're an equal opportunity employer... No need to add here, though, if this is just moving an existing file—I thought this was a new doc. We can handle separately.